test/data/fle2-find/key-document.json

```diff
@@ -0,0 +1,33 @@
+{
+    "status": {
+        "$numberInt": "1"
+    },
+    "_id": {
+        "$binary": {
+            "base64": "YWFhYWFhYWFhYWFhYWFhYQ==",
```

Comment:
Please removing key-document.json. The test uses the keys from `test/data/keys` directory.

Reply:
Gone, and removed the copy from 397 as well.